css/index.css

```diff
@@ -0,0 +1,122 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap");
+@font-face {
+  font-family: Jurassic Park;
+  src: url(/assets/JurassicPark-BL48.ttf);
+}
+
+:root {
+  --primary-color: #00919c;
+  --title-color: #cccccc;
+  --subtitle-color: #808080;
+  --background-color: #101212;
+}
+
+html,
+body,
+h1,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Jurassic Park;
+  font-size: 2rem;
+  letter-spacing: 1px;
+  background-color: var(--background-color);
+}
+
+.header-nav {
+  display: flex;
+  align-items: center;
+  flex-wrap: nowrap;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.version {
+  font-size: 1.5rem;
+  margin-left: 8px;
+  color: var(--subtitle-color);
+}
+
+.header-nav-links {
+  margin-left: auto;
+}
+
+.header-nav-links > nav > ul {
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+  padding-right: 10px;
+  font-size: 2rem;
```

Comment:
have consistency for units

Reply:
Again. 10 being an edge case, I went ahead with px over rem.